integration/token/fungible/sdk/auditor/sdk.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package auditor
+
+import (
+	errors2 "errors"
+
+	"github.com/hyperledger-labs/fabric-smart-client/pkg/node"
+	dig2 "github.com/hyperledger-labs/fabric-smart-client/platform/common/sdk/dig"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/driver"
+	views1 "github.com/hyperledger-labs/fabric-token-sdk/integration/token/common/views"
+	"github.com/hyperledger-labs/fabric-token-sdk/integration/token/fungible/views"
+	tokensdk "github.com/hyperledger-labs/fabric-token-sdk/token/sdk/dig"
+	"github.com/pkg/errors"
+	"go.uber.org/dig"
+)
+
+type SDK struct {
+	dig2.SDK
+}
+
+func NewSDK(registry node.Registry) *SDK {
+	return &SDK{SDK: tokensdk.NewSDK(registry)}
+}
+
+func NewFrom(sdk dig2.SDK) *SDK {
+	return &SDK{SDK: sdk}
+}
+
+func (p *SDK) Install() error {
+	if err := p.SDK.Install(); err != nil {
+		return err
+	}
+
+	if err := p.Container().Invoke(func(in struct {
```

Comment:
it's better to extract this to another function to keep the Install method cleaner:
```p.Container().Invoke(registerViews)```